src/server/game/World/World.cpp

```diff
@@ -3477,3 +3474,13 @@ uint32 World::GetNextWhoListUpdateDelaySecs()
 
     return uint32(ceil(t / 1000.0f));
 }
+
+void World::FinalizePlayerWorldSession(WorldSession* session)
+{
+    uint32 cacheVersion = sWorld->getIntConfig(CONFIG_CLIENTCACHE_VERSION);
```

Comment:
const uint32

Reply:
For the same reason as your other comment this is intentional. Normally I'd agree but i'm explicitly wanting to be able to modify this value from the hook.

Reply:
I see then all should be good.